aiinfra-cluster/installation_scripts/install_cloud_ops_agent.sh

```diff
@@ -43,7 +136,42 @@ handle_debian() {
     }
 
     install_opsagent() {
-        curl -s https://dl.google.com/cloudagents/add-google-cloud-ops-agent-repo.sh | bash -s -- --also-install
+        curl -s https://dl.google.com/cloudagents/add-google-cloud-ops-agent-repo.sh |
+            REPO_SUFFIX=20230214-1.1.1 bash -s -- --also-install
```

Comment:
Should `REPO_SUFFIX` go after `bash -s` ?

Reply:
Nope, it is a variable within the scope of the second subshell within that pipe (rather than an argument to the `bash` command)